docs/tinymist/frontend/neovim.typ

```diff
@@ -22,121 +26,81 @@ See #link("https://github.com/Myriad-Dreamin/tinymist#features")[Tinymist Featur
     },
   }
   ```
-
-== Setup
-
-This section shows you a minimal way to setup tinymist in #link("https://www.lazyvim.org/")[LazyVim]. We also have some tips for users of other distros.
-
-// todo: heading link support
-Please see #md-alter(link(<neovim-extra-settings>)[Extra Settings], () => link("#extra-settings")[Extra Settings]) for more configuration.
-
-=== Setup for #link("https://www.lazyvim.org/")[LazyVim]
-
-Copy or merge the two files to corresponding paths into `~/.config/nvim/`.
-
-- #link("https://github.com/Myriad-Dreamin/tinymist/tree/main/editors/neovim/config/autocmds.lua")[Autocmds file] will help associate the `.typ` file extension with the `typst` filetype.
-- #link("https://github.com/Myriad-Dreamin/tinymist/tree/main/editors/neovim/plugins/tinymist.lua")[Plugin file] will help start tinymist for buffers with the `typst` filetype.
-
-Then, check and restart Neovim.
-
-=== Setup for #link("https://github.com/AstroNvim")[AstroVim]
-
-AstroNvim now uses tinymist by default. Please check the #link("https://github.com/AstroNvim/astrocommunity/tree/main/lua/astrocommunity/pack/typst")[setup script].
-
-=== Setup for #link("https://github.com/neoclide/coc.nvim")[coc.nvim]
-
-You can edit the `coc-settings.json` by executing `:CocConfig`:
-
-```json
-{
-  "languageserver": {
-    "tinymist": {
-      "command": "tinymist",
-      "filetypes": ["typst"],
-      "settings": { ... }
+- Or manually:
+
+ #include "common-finding-executable.typ"
+
+== Configuration
```

Comment:
Some distros has integrated tinymist (typst feature), like AstroVim. Should we mention them here? People who are not familiar with it could be better to use existing integration rather than giving a try with manual configuration.

Reply:
It could be a good idea. Generally established neovim plugins don't mention the different distros in install instructions. There are too many of them, it's more work to follow how they change their configs over time, and most of them use the same installation techniques for plugins. Lazyvim and Packer are the two main package managers that almost any user of a distro will already be familiar with.